### ISSUE TITLE
1.21 Trick Saber Issue

### ISSUE THREAD

**Issue by briamiiller** [Fernthedev/Quest-Trick-Saber#16]
I installed the mod and have the same setting as 1.17.1 and even 1.20.0 but when in game, when I attempt to do tricks, nothing happens, I have tricksaber toggled on and have also reinstalled mod but it won't work.

**Comment by AKASGaming** [comment]
Same with me, only I have the config from 1.19

**Comment by briamiiller** [comment]
Yeah! When you load into other versions of beat saber and have the same mods activated, the data gets carried over and that is what happened to my 1.21 but it just doesn't work. I mean it was only released i believe yesterday so.... but hopefully it gets fixed... i really overuse trick saber......
